modules/@apostrophecms/login/index.js

```diff
@@ -44,6 +44,8 @@ const Promise = require('bluebird');
 const cuid = require('cuid');
 const expressSession = require('express-session');
 
+const loginAttempsNamespace = '@apostrophecms/loginAttempt';
```

Comment:
"Attemps" is still here in the const name

Reply:
It has been removed